lib/viewmodels/forecast_view_model.dart

```diff
@@ -37,18 +37,21 @@ class ForecastViewModel extends BaseModel {
   Future<void> getTodayForecast() async {
     todaysForecast =
         await _forecastService.getForecastForDate(DateTime.now(), position!);
+    notifyListeners();
```

Comment:
Tu chyba niepotrzebne wywołanie. Spowoduje to odświeżenie ekranu a nie ma takiej potrzeby w tym momencie bo i tak po wywołaniu wszystkich 3 funkcji jest setBusy, które też wywoła notifyListeners. Czyli 4 odświeżenia ekranu zamiast jednego.